### PR TITLE
RHEL-69069 Apply clang-format for Balloon Folder 

### PR DESCRIPTION
**Pull request by Jedoku**

### CODE REVIEW

File: Balloon/app/memstat.cpp [hunk 1]
```diff
@@ -13,77 +13,58 @@ CMemStat::~CMemStat()
...
-                             NULL,
-                             RPC_C_AUTHN_LEVEL_PKT,
-                             RPC_C_IMP_LEVEL_IMPERSONATE,
-                             NULL,
-                             EOAC_NONE,
-                             0
-                             );
-
-    if (FAILED(status)) {
+            NULL, -1, NULL, NULL, RPC_C_AUTHN_LEVEL_PKT, RPC_C_IMP_LEVEL_IMPERSONATE, NULL, EOAC_NONE, 0);
```
Comment (kostyanf14) [inline Balloon/app/memstat.cpp:34]: Please check the rules to have one argument in one line

File: Balloon/app/memstat.h [hunk 2]
```diff
@@ -7,25 +7,28 @@
...
     }
 
-    size_t GetSize() {
+    size_t GetSize()
+    {
         return sizeof(m_Stats);
     }
 
-private:
+  private:
```
Comment (kostyanf14) [inline Balloon/app/memstat.h:28]: @YanVugenfirer 

Do we need to align `private/public/protected` in class?

File: Balloon/app/service.cpp [hunk 3]
```diff
@@ -210,71 +224,74 @@ void CService::ServiceMain(DWORD argc, LPTSTR *argv)
...
+                                                  CM_NOTIFY_ACTION Action,
+                                                  PCM_NOTIFY_EVENT_DATA EventData,
+                                                  DWORD EventDataSize)
 {
     CService *pThis = reinterpret_cast<CService *>(Context);
     DWORD event = 0;
 
     switch (Action)
     {
-        case CM_NOTIFY_ACTION_DEVICEINTERFACEARRIVAL:
```
Comment (kostyanf14) [inline Balloon/app/service.cpp:260]: @YanVugenfirer 

Should `case` have an extra indentation?

**Comment by kostyanf14** [comment]
@Jedoku `RHEL-65783` jira tag should be first!

File: Balloon/app/memstat.cpp (hunk 1 above)
Comment (kostyanf14) [reply Balloon/app/memstat.cpp]: Done

File: Balloon/app/service.cpp (hunk 3 above)
Comment (kostyanf14) [reply Balloon/app/service.cpp]: Done

File: Balloon/sys/public.h [hunk 4]
```diff
@@ -3,28 +3,28 @@
...
-#define VIRTIO_BALLOON_S_MINFLT   3   /* Number of minor faults */
-#define VIRTIO_BALLOON_S_MEMFREE  4   /* Total amount of free memory */
-#define VIRTIO_BALLOON_S_MEMTOT   5   /* Total amount of memory */
-#define VIRTIO_BALLOON_S_AVAIL    6   /* Available memory */
-#define VIRTIO_BALLOON_S_CACHES   7   /* Disk caches */
-#define VIRTIO_BALLOON_S_HTLB_PGALLOC  8  /* Hugetlb page allocations */
-#define VIRTIO_BALLOON_S_HTLB_PGFAIL   9  /* Hugetlb page allocation failures */
-#define VIRTIO_BALLOON_S_NR       10
+#define VIRTIO_BALLOON_S_SWAP_IN 0      /* Amount of memory swapped in */
+#define VIRTIO_BALLOON_S_SWAP_OUT 1     /* Amount of memory swapped out */
```
Comment (YanVugenfirer) [inline Balloon/sys/public.h:9]: Why here the defines are not aligned?

File: Balloon/sys/trace.h [hunk 5]
```diff
@@ -36,86 +36,84 @@ extern ULONG driverDebugFlags;
...
-  #define TRACE_LEVEL_WARNING     3
-  #define TRACE_LEVEL_INFORMATION 4
-  #define TRACE_LEVEL_VERBOSE     5
-  #define TRACE_LEVEL_RESERVED6   6
-  #define TRACE_LEVEL_RESERVED7   7
-  #define TRACE_LEVEL_RESERVED8   8
-  #define TRACE_LEVEL_RESERVED9   9
+#define TRACE_LEVEL_NONE 0
+#define TRACE_LEVEL_FATAL 1
+#define TRACE_LEVEL_ERROR 2
```
Comment (YanVugenfirer) [inline Balloon/sys/trace.h:41]: Actually should be aligned

File: Balloon/sys/trace.h [hunk 6]
```diff
@@ -36,86 +36,84 @@ extern ULONG driverDebugFlags;
...
-if (level > driverDebugLevel || !bDebugPrint || !(driverDebugFlags & flags)) {} \
-else VirtioDebugPrintProc(message, __VA_ARGS__)
-
-#define WPP_INIT_TRACING(a,b)
+#define DBG_INIT 0x00000001
+#define DBG_PNP 0x00000002
+#define DBG_POWER 0x00000004
+#define DBG_WMI 0x00000008
+#define DBG_CREATE_CLOSE 0x00000010
+#define DBG_IOCTLS 0x00000020
```
Comment (YanVugenfirer) [inline Balloon/sys/trace.h:59]: Same, missing alignment

File: Balloon/app/device.cpp [hunk 7]
```diff
@@ -201,8 +220,8 @@ PTCHAR CDevice::GetDevicePath(IN LPGUID InterfaceGuid)
 
     do
     {
-        cr = CM_Get_Device_Interface_List_Size(&DeviceInterfaceListLength,
-            InterfaceGuid, NULL, CM_GET_DEVICE_INTERFACE_LIST_PRESENT);
+        cr = CM_Get_Device_Interface_List_Size(
+                                 &DeviceInterfaceListLength, InterfaceGuid, NULL, CM_GET_DEVICE_INTERFACE_LIST_PRESENT);
```
Comment (YanVugenfirer) [inline Balloon/app/device.cpp:224]: Can we not skip the line in this case? If the line too long, the parameters should be slip with the usual rule of parameter per line

File: Balloon/app/device.cpp [hunk 8]
```diff
@@ -214,18 +233,20 @@ PTCHAR CDevice::GetDevicePath(IN LPGUID InterfaceGuid)
             HeapFree(GetProcessHeap(), 0, DeviceInterfaceList);
         }
 
-        DeviceInterfaceList = (PWSTR)HeapAlloc(GetProcessHeap(),
-            HEAP_ZERO_MEMORY, DeviceInterfaceListLength * sizeof(WCHAR));
+        DeviceInterfaceList = (PWSTR)HeapAlloc(
+                                 GetProcessHeap(), HEAP_ZERO_MEMORY, DeviceInterfaceListLength * sizeof(WCHAR));
```
Comment (YanVugenfirer) [inline Balloon/app/device.cpp:237]: Can we not skip the line in this case? If the line too long, the parameters should be slip with the usual rule of parameter per line

**Comment by kostyanf14** [comment]
rerun tests

**Review by YanVugenfirer — APPROVED** [review]

**Pull request merged**